samtools/1.16.1/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:focal as app
+
+ARG SAMTOOLSVER="1.16.1"
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="Samtools"
+LABEL software.version="${SAMTOOLSVER}"
+LABEL description="Tools (written in C using htslib) for manipulating next-generation sequencing data"
+LABEL website="https://github.com/samtools/samtools"
+LABEL license="https://github.com/samtools/samtools/blob/develop/LICENSE"
+LABEL maintainer1="Shelby Bennett"
+LABEL maintainer1.email="shelby.bennett@dgs.virginia.gov"
```

Comment:
Can you change `maintainer1` and `maintainer1.email` to `maintainer` and `maintainer.email`?